### ISSUE TITLE
Timestamps on files in light mode are too low contrast

### ISSUE THREAD

**Issue by robintown** [vector-im/element-web#20674]
### Steps to reproduce

1. Use bubble layout in light mode
2. Hover over any file

### Outcome

![timestamp](https://user-images.githubusercontent.com/48614497/150575976-1817bc27-ca9c-4559-91e6-3daee099a921.png)

#### What did you expect?

The timestamp should be easily readable.

#### What happened instead?

The timestamp is white on a light gray background, which provides poor contrast.

### Operating system

NixOS unstable

### Browser information

Firefox 96.0

### URL for webapp

develop.element.io

### Application version

Element version: 06fa4f4440a4-react-010cbadc8ecf-js-230e3b4acee3 Olm version: 3.2.8

### Homeserver

townsendandsmith.ml

### Will you send logs?

No
